### PR TITLE
Handle when v1 is deprecated, print a message instead of an error

### PR DESCRIPTION
**Pull request by JuneKelly**
Partial fix for https://github.com/overleaf/sharelatex/issues/1401

### CODE REVIEW

File: src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java [hunk 1]
```diff
@@ -18,6 +18,13 @@
         "see https://www.overleaf.com/help/342 for more information."
     );
 
+    public static final List<String> OVERLEAF_V1_DEPRECATED_REASON = Arrays.asList(
+            "Overleaf v1 is deprecated, and you need to migrate this project to v2.",
+            "",
+            "If this is unexpected, please contact us at support@overleaf.com, or",
+            "see https://www.overleaf.com/help/342 for more information."
```
Comment (JuneKelly) [inline src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java]: This wording is bad. 

Suggestions @jdleesmiller and @jpallen ?

Comment (jpallen) [reply src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java]: How about:

> This project needs to be migrated to Overleaf v2. Please visit the project online to do this:
> 
> https://www.overleaf.com/<project_id>
>
> After migrating this project to Overleaf v2, you will be prompted to update your git remote to the new Overleaf v2 identifier.
>
> Please see https://www.overleaf.com/help/342 for more information, or contact us at support@overleaf.com.

Comment (jdleesmiller) [reply src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java]: We could mirror the wording from https://github.com/sharelatex/web-sharelatex/blob/master/app/views/project/v2-import.pug .

> This project has not yet been moved into the new version of Overleaf. You will need to move it in order to continue working on it. Please visit the project online to do this:

Ah, James beat me to it. James's wording also looks good to me.

Comment (jpallen) [reply src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java]: I like John's better (consistent and less harsh), so the whole thing would be:

> This project has not yet been moved into the new version of Overleaf. You will need to move it in order to continue working on it.  Please visit this project online to do this:
> 
> https://www.overleaf.com/<project_id>
> 
> After migrating this project to the new version of Overleaf, you will be prompted to update your git remote to the project's new identifier.
> 
> Please see https://www.overleaf.com/help/342 for more information, or contact us at support@overleaf.com.

**Review by jpallen — APPROVED** [review]
LGTM, subject to updating the wording as in the comment

**Comment by JuneKelly** [comment]
Example output:

```
shanek@sk-fed:~/t/g/321hpcdtydwwqfh|master✓
➤ git pull
remote: This project has not yet been moved into the new version of Overleaf. You will
remote: need to move it in order to continue working on it. Please visit this project
remote: online to do this:
remote: 
remote:     http://v2.overleaf.test:4000/321hpcdtydwwqfh
remote: 
remote: After migrating this project to the new version of Overleaf, you will be
remote: prompted to update your git remote to the project's new identifier.
remote: 
remote: If this is unexpected, please contact us at support@overleaf.com, or
remote: see https://www.overleaf.com/help/342 for more information.
fatal: repository 'http://localhost:8000/321hpcdtydwwqfh/' not found
```

**Review by jpallen — APPROVED** [review]

**Pull request merged**